src/Shared/Program.cs

```diff
@@ -40,7 +42,11 @@ public static void Main(string[] args)
             {
                 input = ParseListingName(input);
 
-                Type? target = assembly.GetTypes().FirstOrDefault(type => type.FullName.Contains(input));
+                Type? target = assembly.GetTypes().FirstOrDefault(type =>
+                {
+                    Regex reg = new Regex($"{input}\\.+");
```

Comment:
This will create the regex for each Type declared in the assembly. It is better to move it outside of the FirstOrDefault so it is only created once.